x-pack/plugins/beats_management/server/lib/domains/__tests__/beats/assign_tags.test.ts

```diff
@@ -72,14 +80,17 @@ describe('Beats Domain Lib', () => {
         {
           configuration_blocks: [],
           id: 'production',
+          last_updated: new Date(),
```

Comment:
Not having a passing CI has made me more passive about testing than I should be. These tests were broken by my last changes. Have we documented a standard testing procedure before merging code?

Reply:
in the readme there is the command for running tests